app/routes/webhooks.py

```
@@ -14,74 +17,74 @@
-        return APIResponse.error(
-            status_code=status.HTTP_400_BAD_REQUEST,
-            message=constants.INVALID_WEBHOOK_SIGNATURE,
-        )
-
-    except Exception as e:
-        logger.error(f"Unexpected error processing webhook: {str(e)}")
-        return APIResponse.error(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            message=constants.SERVICE_UNAVAILABLE,
-        )
+	"""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)